gapic-generator/lib/google/gapic/generators/default_generator.rb

```diff
@@ -49,36 +49,12 @@ def generate
           files = []
 
           api_services(@api).each do |service|
-            files << cop(gen("client.erb", ruby_file_path(service),
-                             api: @api, service: service))
+            files << g("client.erb", ruby_file_path(service),
+                       api: @api, service: service, format: true)
```

Comment:
Why is `format: true` passed here? Isn't `true` the default?

Reply:
No, it shouldn’t be the default

Reply:
@quartzmo I'm rethinking this. Perhaps formatting the output should be default, and users should pass `format: false` to opt-out. Thoughts?

Reply:
Isn't the method [defined here](https://github.com/googleapis/gapic-generator-ruby/blob/9c0a8a3d420925b60ad136019844f672f1a1a224/gapic-generator/lib/google/gapic/generators/base_generator.rb#L72) with `format: true`?

Reply:
Yes, I think formatting the output should be default, and users should pass `format: false` to opt-out.

Reply:
kk, i'll change it.